utils/ftp.py

```diff
@@ -301,9 +303,11 @@ def update_time_difference(self):
             if name == TIMECHECK_FILE_NAME:
                 self.dt = now - time
                 self.dele(TIMECHECK_FILE_NAME)
+                self.cwd("/")
```

Comment:
Actually I suggested that since I considered `cd'ing into a known upload directory after successful connection to the server` an unexpected behavior.